intellij/src/nl/tudelft/watchdog/intellij/logic/ui/listeners/EditorListener.java

```diff
@@ -86,29 +85,29 @@ public void caretAdded(CaretEvent e) {
             public void caretRemoved(CaretEvent e) {
                 // Intentionally left blank
             }
-		};
-		editor.getCaretModel().addCaretListener(caretListener);
+        };
+        editor.getCaretModel().addCaretListener(caretListener);
 
-		// creates a listener for redraws of the view, e.g. when scrolled
-		visibleAreaListener = new VisibleAreaListener() {
+        // creates a listener for redraws of the view, e.g. when scrolled
+        visibleAreaListener = new VisibleAreaListener() {
             @Override
             public void visibleAreaChanged(VisibleAreaEvent e) {
                 if(e.getEditor().isViewer()) {
                     eventManager.update(new EditorEvent(editor, EventType.PAINT));
                 }
             }
         };
-		editor.getScrollingModel().addVisibleAreaListener(visibleAreaListener);
+        editor.getScrollingModel().addVisibleAreaListener(visibleAreaListener);
 
-	}
+    }
 
-	@Override
-	public void dispose() {
+    /** Removes all listeners registered with this editor. */
+    public void removeListeners() {
```

Comment:
When and how is this called now?

Reply:
Hm, I have reverted these changes, but it appears to be confusing with my whitespace changes. Apparently my editor is configured to use spaces, but the files use tabs. In https://github.com/TestRoots/watchdog/pull/274/files?w=1#diff-1b718277d0fe136c9530faa96dd1367e you can see that there were only whitespace changes in this file.